.codespell/.codespellrc

```diff
@@ -0,0 +1,5 @@
+[codespell]
+skip = .git,node_modules,artifacts-zk,yarn.lock,contracts-preprocessed,Cargo.lock
+count =
```

Comment:
I am not knowledgable about codespell, but it looks like a field that should be filled

Reply:
Fixed